CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+
+# Contributing
+
+<a name="development" id="development"></a>
+## Development and contributing setup
+
+These instructions explain how to develop the cloudgov style while using it on another (companion) site. This allows to see the result of the style changes on an actual site.
+
+0. Ensure node/npm is installed. To ensure you're using the most recent version of node that this project supports, use [nvm](https://github.com/creationix/nvm).
+0. Run `git clone git@github.com:18F/cg-style.git cloudgov-style` if you haven't done so already.
+0. Enter the project directory with `cd cloudgov-style`.
+0. If using **nvm**, run `nvm use`. Otherwise, try and install the node version that is in `.nvmrc`.
+0. Run `npm install` in the `cloudgov-style` repo.
+0. Run `npm run build` in the `cloudgov-style` repo.
+0. In the *cloudgov-style* repo, link the current module by running `npm link` in the root of the repo.
```

Comment:
I might re-phrase this as
> In the *cloudgov-style* repo, make the module linkable by running `npm link` in the root of the repo.

I think it's good to have a distinction in terminology when talking about the app being linked vs app linking from.